File: practices/fizzbuzz.py
#EM 7th fizzbuzz 
x = 1
while x <= 50:
        if x % 3 ==0 and x % 5 ==0:
               print("FizzBuzz")
        elif x % 5 ==0:
               print("Buzz!")
        elif x % 3 ==0:
               print("Fizz!")
        else:
               print(x)
        x+=1
print("Thats all numbers through 50")

File: c_programming_2/practice/fizzbuzz.c
// em 7th fizzbuzz
#include <stdio.h>

int main (void){
    int num = 1;

    for(int num=1; num<=50; num++){
        if(num % 3 ==0){
            printf("fizz\n");
        }
        else if(num % 5 == 0){
            printf("buzz\n");
        }
        else if(num % 3 == 0 && num % 5 ==0){
            printf("fizzbuzz\n");
        }
       } 
       num++;

    return 0; 
}